### ISSUE TITLE
Crashed ru.yandex.mobile.YandexMobileMetrica.YMMAsyncExecutor.ivarQueue 2.9.1+ios11beta7

### ISSUE THREAD

**Issue by alexsanyo** [yandexmobile/metrica-sdk-ios#64]
```
Thread 12 Queue : ru.yandex.mobile.YandexMobileMetrica.YMMAsyncExecutor.ivarQueue (serial)
#0	0x11722c553 in CFEqual ()
#1	0x60800043f0a0 in 0x60800043f0a0 ()
#2	0x11725e18e in CFBundleCreate ()
#3	0x10d4dcee5 in -[YMMFramework initWithBundleAtPath:] ()
#4	0x10d4edec9 in +[YMMMetricaDynamicFrameworks mcServices] ()
#5	0x10d4c52ce in +[YMMBundleProvider allItems] ()
#6	0x10d4e1dae in -[YMMIdentityReporter readyToReport] ()
#7	0x10d4e1f3d in -[YMMIdentityReporter reportIfNeeded] ()
#8	0x10d4e2223 in -[YMMIdentityReporterContainer reportIfNeeded] ()
#9	0x10d523299 in __48-[YMMYandexMetricaImpl didAddEventNotification:]_block_invoke ()
#10	0x11856d293 in _dispatch_call_block_and_release ()
#11	0x11856e2d5 in _dispatch_client_callout ()
#12	0x118575dd1 in _dispatch_queue_serial_drain ()
#13	0x118576622 in _dispatch_queue_invoke ()
#14	0x118572d74 in _dispatch_queue_override_invoke ()
#15	0x1185789ed in _dispatch_root_queue_drain ()
#16	0x118578757 in _dispatch_worker_thread3 ()
#17	0x1189d95a2 in _pthread_wqthread ()
#18	0x1189d907d in start_wqthread ()
```

**Comment by yvasileuski** [comment]
Looks like https://github.com/yandexmobile/metrica-sdk-ios/issues/63

Could you please specify version of AppMetrica SDK used or link to crash report in AppMetrica Web interface?

**Comment by alexsanyo** [comment]
Podfile.lock:
```
  - YandexMobileAds (2.7.1):
    - YandexMobileAds/Static (= 2.7.1)
  - YandexMobileAds/Static (2.7.1):
    - YandexMobileMetrica (< 3.0.0, >= 2.8.0)
  - YandexMobileMetrica (2.9.1):
    - YandexMobileMetrica/Static (= 2.9.1)
  - YandexMobileMetrica/Static (2.9.1
```
The crash seems to happen only on ios11

**Comment by fallkrum** [comment]
Could you provide information on how often it crashes on iOS 11?

**Comment by alexsanyo** [comment]
The app uses Yandex mobile ads and works fine on iOS 9,10.
On iOS11 beta 7(and earlier) the app constantly crashes a few days after initial installation.
This code leads to crash:
`        [YMMYandexMetrica activateWithApiKey:MYKEY];
`

**Comment by fallkrum** [comment]
Thanks a lot for information, the issue will be analysed and fixed as soon as possible.

**Comment by fallkrum** [comment]
Fixed in 2.9.4
